repairnator/repairnator-dockerpool/src/main/java/fr/inria/spirals/repairnator/dockerpool/RunnablePipelineContainer.java

```diff
@@ -70,9 +74,13 @@ public void run() {
             LOGGER.info("The container has finished with status code: "+exitStatus.statusCode());
             docker.removeContainer(container.id());
             Launcher.runningDockerContainer.remove(container.id());
+
+            googleSpreadSheetTreatedBuildTracking.setStatus("TREATED");
         } catch (InterruptedException|DockerException e) {
             LOGGER.error("Error while creating/running the container for build id "+buildId, e);
+            googleSpreadSheetTreatedBuildTracking.setStatus("INTERRUPTED");
```

Comment:
I'm not sure that I can do that in the current way of the serializer in the code. I have to think about it.